### DISCUSSION TITLE
Using Mailto: for blog post replies

### DISCUSSION THREAD

**Discussion by arcadiapage** [community/community#46639] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm working on adding a mailto: link to my blog post layout so that readers can reply by email.  The issue I'm having is that I would like the title of the blog post to be automatically included in the subject line when someone clicks the mailto: link. 

This can be done with 11ty. [Here's the tutorial I was following](https://www.lkhrs.com/blog/2021/07/alternative-to-comments/
).
I was wondering if the same can be done with Jekyll. My code right now is like this, but it doesn't work:

```
<div class="text-center py-10">
    <p>Have some thoughts on this post?
    <a class="button" href="mailto:name@myemailaddress?subject=Re:{{ post.title }}">Reply with an email</a></p>
</div>
```

All it gives me is the subject line is the Re: and the rest is blank. Any suggestions would be appreciated!

**Comment by yoannchaudet** [comment]
You can use `page.title`. See more here https://jekyllrb.com/docs/variables/#page-variables.

**Reply by arcadiapage** [reply]
Perfect! Thank you!
